src/main/java/kayu/commands/EventCommand.java

```diff
@@ -0,0 +1,49 @@
+package kayu.commands;
+
+import static kayu.commands.CommandMessage.MESSAGE_CREATED_EVENT;
+import static kayu.commands.CommandType.EVENT;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+
+import kayu.exception.DukeException;
+import kayu.parser.DateTimeFormat;
+import kayu.service.TaskList;
+import kayu.task.Event;
+import kayu.task.Task;
+
+/**
+ * Represents an {@link kayu.commands.AddTaskCommand} that creates a {@link kayu.task.Event}
+ * and saves it in {@link kayu.service.TaskList}.
+ */
+public class EventCommand extends AddTaskCommand {
+
+    /** Key word for command. */
+    public static final String COMMAND_WORD = "event";
+
+    /**
+     * Initializes a Event- {@link kayu.commands.AddTaskCommand}.
```

Comment:
Similar Grammatical error here for "a" vs "an". Might want to check through your entire Javadoc.